src/test/java/lotto/domain/AutoLottoMachineTest.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class AutoLottoMachineTest {
+    private AutoLottoMachine autoLottoMachine = new AutoLottoMachine();
+
+    @DisplayName("목표 장수에 맞추어 티켓을 생성하는지 확인")
+    @Test
+    void 목표에_맞추어_티켓을_생성한다(){
```

Comment:
```@DisplayName``` 이 있으니 메서드 이름은 한글이 아니라도 괜찮을 거 같네요! 
```suggestion
    @DisplayName("목표 장수에 맞추어 티켓을 생성하는지 확인")
    @Test
    void 목표에_맞추어_티켓을_생성한다() {
```

작성하신 코드 전체적으로 공백 컨벤션 확인 부탁드려요!

Reply:
공백 컨벤션 확인했습니다